DEVELOPMENT.md

```diff
@@ -0,0 +1,82 @@
+
+We are currently using two sets of these as examples:
+
+> **Note**
+>
+> 1. [Example TensorFlow model from the MLflow GitHub repository](https://github.com/mlflow/mlflow/tree/ef0f922ca874dd94f9b88e8bfbde2a8cb2c385d2/examples/tensorflow)
+>
+> 2. [Example AzureML + MLflow + sklearn model provided by](https://github.com/stefan-bergstein/azureml-model-to-edge/tree/8e41fbfb2991cf437a6477553939e04c71103555/models/bike-rentals-auto-ml) [@stefan-bergstein](https://github.com/stefan-bergstein)
+>
+> See [#25](https://github.com/opendatahub-io/ai-edge/issues/25) and [#27](https://github.com/opendatahub-io/ai-edge/issues/27) for why.
+
+|                       | **1**                    | **2**                  |
+|-----------------------|--------------------------|------------------------|
+| **Model (Pipelines)** | `tensorflow-housing`     | `bike-rentals-auto-ml` |
+| **App (ACM)**         | `tensorflow-housing-app` | `bike-rental-app`      |
+
+### Clusters
+
+> **Note**
+> See *"Infrastructure Configuration"* in [README.md](./README.md) for more information.
+
+You can visualize the clusters like this:
+
+* `local-cluster` (default): This is the Hub cluster, might be located in a Core Data Center. Good connection and not as restrictive as the others.
+
+* `near-edge-*`: These clusters might be located at a factory. Restricted firewall and high emphasis on security.
+
+We use a mix of AWS and GCP to host these clusters for diversity. Most of your dev work will be done on `local-cluster`, while ACM (also on `local-cluster`) deploys apps to `near-edge-*` clusters.
+
+## Pipelines
+
+> **Note**
+> If you’re not going to be pushing new images to Quay, you can use the existing ones from the [`rhoai-edge` Quay organization](https://quay.io/organization/rhoai-edge).
+
+* Create a new namespace for testing, preferably with the following name: `<your-kerberos-name>-pipeline-dev`
+* Follow the steps in the [`Pipelines Setup README`](./pipelines/README.md).
+
+## ACM
+
+> **Note**
+> These instructions will assume that you are making changes to the existing `bike-rental-app`.
+
+* Create a new testing branch on your fork of the `opendatahub-io/ai-edge` repo, preferably named `<your-kerberos-name>-acm-dev`.
+* Update and push these files (please see [this example](https://github.com/grdryn/ai-edge/commit/39a89e5bcacb6ae5a5b649797e2341889a94ea42)):
+  * `acm/registration/near-edge/overlays/bike-rental-app/kustomization.yaml`
+    * Change the namespace, Git path, and channel patch to your fork.
+  * `acm/registration/near-edge/base/near-edge.yaml`
+    * Change the Git branch **and namespace** to your dev one.
+  * `acm/odh-edge/apps/bike-rental-app/kustomization.yaml`
+    * Change the namespace to your own dev one.
+* `oc apply -k acm/registration`
+* Add your dev namespace to the near-edge Cluster Set's `Namespace Bindings`.
```

Comment:
These instructions can be updated once Ger merges his PR #76 in a week or two (hopefully) but this works for now.